source/AI.cpp

```diff
@@ -3514,13 +3514,40 @@ void AI::MovePlayer(Ship &ship, const PlayerInfo &player, Command &activeCommand
 	AimTurrets(ship, firingCommands, !Preferences::Has("Turrets focus fire"));
 	if(Preferences::Has("Automatic firing") && !ship.IsBoarding()
 			&& !(autoPilot | activeCommands).Has(Command::LAND | Command::JUMP | Command::FLEET_JUMP | Command::BOARD)
-			&& (!target || target->GetGovernment()->IsEnemy()))
+			&& (!target || target->GetGovernment()->IsEnemy())
+			&& !isMouseTurningEnabled)
 		AutoFire(ship, firingCommands, false);
+
+	if (activeCommands.Has(Command::MOUSETURNING)) {
+		isMouseTurningEnabled = !isMouseTurningEnabled;
```

Comment:
This is no longer required if you rely on the preference `Preferences::Has("alt-mouse turning")` which I introduced in my feedback pull request to your fork.  https://github.com/CAPTAIN1947/endless-sky/pull/1